### DISCUSSION TITLE
Professional services?

### DISCUSSION THREAD

**Discussion by djMax** [chakra-ui/panda#1672] [Q&A]
Would the team consider professional services or "funded" Panda feature development?

**Comment by segunadebayo** [comment]
Hi @djMax, that sounds like an interesting idea. What do you have in mind?

**Reply by djMax** [reply]
I don't have a well formed plan, but we are using panda for a rewrite of our UI kit and have some not-unique needs like:

* Proper packaging for consumption as a library rather than an app
* Build time improvements around tree shaking (and not tree shaking) and clean style/recipe patterns in our components

So some of it would just be open conversations in github here but with some funding to give them some attention, and some might be paying to prioritize features we need but that are still useful to others and/or agreed upon as good things to do.
